file-formats/nont/examples/z_aff_example_nont.nont.json

```diff
@@ -0,0 +1,11 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "Example SAP Object Node Type.",
+    "originalLanguage": "en"
+  },
+  "name": "Z_Aff_Example_Nont",
+  "expandedName": "Z_Aff_Example_Nont_Expanded",
+  "parentName": "Z_AFF_EXAMPLE_RONT",
```

Comment:
Question: Would it make sense to use the name with camel cases or even the expanded name here? Did you discuss this?

Reply:
We have just discussed this and we really like the idea to display the name of the referenced SAP Object Type in camel case (rather than upper case). Thanks for the very good suggestions!